lib/cisco_node_utils/cmd_ref/tacacs_server_host.yaml

```diff
@@ -1,37 +1,66 @@
 # tacacs_server_host
 ---
-_exclude: [ios_xr]
+_template:
+  nexus:
+    get_command: "show run tacacs all"
+  ios_xr:
+    get_command: "show running-config tacacs-server"
 
 encryption:
-  set_value: "%s tacacs-server host %s key %s %s"
+  nexus:
+    set_value: "<state> tacacs-server host <ip> key <key>"
```

Comment:
The `key` vars changed from `key %s %s` to `key <key>`. Do we not need the second var anymore? 

The old setter used to pass `encryption_password` there:

```
          config_set('tacacs_server_host', 'encryption', 'no', @name,
                     encryption_type,
                     encryption_password)
```